### ISSUE TITLE
Book 05: Sentence 030

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#29]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/dc8de9e7-5c35-4db0-8461-8944acd9859c)

```
# sent_id = Book_05_-_The_Omens.txt:30
# text = Inin, ioaltepuztli, iuh mjtoa, qujlmach iehoatl, ipan mjxeoaia, in tezcatlipuca: injc teca mocacaiaoaia, teca maujltiaia, qujtoa.
# text[norm] = Inin, yohualtepoztli, iuh mihtoa, quilmach yehhuatl, ipan mixehuaya, in Tezcatlipoca: inic teca mocacayahuaya, teca mahuiltiaya, quihtoa.
# text[orig] = In·in·,·io·altepuztli·,·iuh·mjtoa·,·qujlmach¶iehoatl·,·ipan·mjxeoaia·,·in·tezca¶tlipuca·:·injc·teca·mocacaiaoaia·,·teca·maujltiaia·,·qujtoa·.
# text[osp] = Y cuando oyan golpes como de qujen hiende madero con hacha, lo qual de noche suena lexos: espantauan se de aquellos golpes, y tomauan mal aguero: dezian. Que estos golpes: eran illusion de Tezcatlipuca, conque espantaua de noche, y burlaua a los que andan de noche.
# references = A&D/165:3
1	Inin	inin	PRON	_	_	12	dislocated	_	Orig=In·in|Folio=4v,4v|Paragraph=4,4|Line=12,12|Norm=Inin
2	,	,	PUNCT	_		3	punct	_	Folio=4v|Paragraph=4|Line=12|Norm=,|Analysed=Yes
3	ioaltepuztli	yohualtepoztli	NOUN	_	State=Abs	1	_	_	Orig=io·altepuztli|Folio=4v,4v|Paragraph=4,4|Line=13,13|Norm=yohualtepoztli|Split=yohual·tepoztli|Analysed=Yes
4	,	,	PUNCT	_		6	punct	_	Folio=4v|Paragraph=4|Line=13|Norm=,|Analysed=Yes
5	iuh	iuh	ADV	_		6	_	_	Folio=4v|Paragraph=4|Line=13|Norm=iuh|AmbigNorm=True|Override=False|Gloss=así|Analysed=Yes
6	mjtoa	ihtoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	12	_	_	Folio=4v|Paragraph=4|Line=13|Norm=mihtoa|Analysed=Yes
7	,	,	PUNCT	_		12	punct	_	Folio=4v|Paragraph=4|Line=13|Norm=,|Analysed=Yes
8	qujlmach	quilmach	ADV	_	_	12	_	_	Folio=4v|Paragraph=4|Line=13|Norm=quilmach|Gloss=al.parecer
9	iehoatl	yehhuatl	PRON	_	Number=Sing|Person=3|PronType=Prs	12	_	_	Folio=5r|Paragraph=1|Line=1|Norm=yehhuatl|Gloss=él|Analysed=Yes
10	,	,	PUNCT	_		12	punct	_	Folio=5r|Paragraph=1|Line=1|Norm=,|Analysed=Yes
11	ipan	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	12	_	_	Folio=5r|Paragraph=1|Line=1|Norm=ipan|Analysed=Yes
12	mjxeoaia	ixehua	VERB	_	Aspect=Imp|Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Past|VerbForm=Fin	0	root	_	Folio=5r|Paragraph=1|Line=1|Norm=mixehuaya|Analysed=Yes
13	,	,	PUNCT	_		15	punct	_	Folio=5r|Paragraph=1|Line=1|Norm=,|Analysed=Yes
14	in	in	DET	_	_	15	_	_	Folio=5r|Paragraph=1|Line=1|Norm=in
15	tezcatlipuca	Tezcatlipoca	PROPN	_		11	_	_	Orig=tezca¶·tlipuca|Folio=5r,5r|Paragraph=1,1|Line=1,2|Norm=Tezcatlipoca|Analysed=Yes
16	:	:	PUNCT	_		19	punct	_	Folio=5r|Paragraph=1|Line=2|Norm=:|Analysed=Yes
17	injc	inic	SCONJ	_		19	_	_	Folio=5r|Paragraph=1|Line=2|Norm=inic|Analysed=Yes
18	teca	ca	NOUN	_	Animacy[psor]=Anim|NounType=Relat|Person[psor]=3|State=Pos	19	_	_	Folio=5r|Paragraph=1|Line=2|Norm=teca|Analysed=Yes
19	mocacaiaoaia	cacayahua	VERB	_	Aspect=Imp|Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Past|VerbForm=Fin	12	_	_	Folio=5r|Paragraph=1|Line=2|Norm=mocacayahuaya|Analysed=Yes
20	,	,	PUNCT	_		22	punct	_	Folio=5r|Paragraph=1|Line=2|Norm=,|Analysed=Yes
21	teca	ca	NOUN	_	Animacy[psor]=Anim|NounType=Relat|Person[psor]=3|State=Pos	22	_	_	Folio=5r|Paragraph=1|Line=3|Norm=teca|Analysed=Yes
22	maujltiaia	mahuiltia	VERB	_	Aspect=Imp|Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Past|VerbForm=Fin	19	_	_Folio=5r|Paragraph=1|Line=3|Norm=mahuiltiaya|Analysed=Yes
23	,	,	PUNCT	_		24	punct	_	Folio=5r|Paragraph=1|Line=3|Norm=,|Analysed=Yes
24	qujtoa	ihtoa	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	12	__	Folio=5r|Paragraph=1|Line=3|Norm=quihtoa|Analysed=Yes
25	.	.	PUNCT	_		12	punct	_	Folio=5r|Paragraph=1|Line=3|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
López Austin:
> Así se decía que el hacha nocturna dizque era él, dizque repre­sentaba a Tezcatlipoca, que se burlaba de la gente y la espantaba.

Questions:
- What is *inin*, wha does it relate to?
- Is *teca* relational with *te-* possessive?

**Comment by Lguyogiro** [comment]
Here is an initial proposal:

![imagen](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/a6d3a256-17c6-4457-a7b4-aef9e09b82c1)


> What is inin, wha does it relate to?

(_inin_: "this") I think calling this the dislocated subject of _mixehuaya_ makes sense. 

> Is teca relational with te- possessive?

It looks that way to me.

one weird thing about this is that there is _mihtoa_ introducing the clause about how the night axe represents tezcatlipoca, and another _quihtoa_ at the end that, under my analysis, introduces the other clauses describing details about the night axe. But I'm not sure if it is the night axe that se burlaba de la gente or if it is tezcatlipoca and the two subsequent clauses are _acl_ to Tezcatlipoca.

**Comment by ftyers** [comment]
How about making *Inin, yohualtepoztli* be the `nsubj` of *mihtoa*? 

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/4c93a836-6764-44ca-8e62-185d7053c11e)
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/5f916237-67da-4ca0-aaf0-dad618149c9a)
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/8624a118-aafb-4998-a5e8-a561fb87435e)
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/e52c5254-36f4-496e-a371-8cf957c27856)

Dislocated is a bit *hmm*, perhaps we can look at some other examples of *Inin ...*:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/92eff08b-e70b-4ca8-9a97-a7063fd9ba2a)

**Comment by Lguyogiro** [comment]
> How about making Inin, yohualtepoztli be the nsubj of mihtoa?

I would argue this is much more _hmmm_, since this would be equivalent to a sentence like "the night axe is said," which I don't think makes much sense. Inspired by this suggestion, though, we could say that _inin, yohualtepoztli_ is a copular clause ("it is the night axe, (it) is said, it is what represents tezcatlipoca..."). so something like the following:

![imagen](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/25f3e313-1dbf-46c8-a935-5695c195119e)

Alternatively, the _mijtoa_ could be treated as interrupting reported speech (as per the parataxis guidelines?)

**Comment by ftyers** [comment]
> > How about making Inin, yohualtepoztli be the nsubj of mihtoa?
> 
> I would argue this is much more _hmmm_, since this would be equivalent to a sentence like "the night axe is said," which I don't think makes much sense. Inspired by this suggestion, though, we could say that _inin, yohualtepoztli_ is a copular clause ("it is the night axe, (it) is said, it is what represents tezcatlipoca..."). so something like the following:
> 

Yeah, agree, especially with that construction.

*It is said (that) it is the night axe, allegedly it represents Tezcatlipoca*

Something like this...

> Alternatively, the _mijtoa_ could be treated as interrupting reported speech (as per the parataxis guidelines?)

I don't like that so much when it's impersonal with *mo-*

**Comment by Lguyogiro** [comment]
LGTM

re `parataxis:shared`, yeah this seems to get a little challenging....e.g. are there iSlAnD cOnStrAiNtS on parataxis:shared? haha, i.e. if the clause has an embedded clause with arg 1, and there is parataxis between the main clause and another clause that also has arg1 as a core argument....to we still consider this shared? probably not. in which case none of these are shared.

**Comment by ftyers** [comment]
Yeah, I don't know, I figure if any of the arguments of the main clauses are shared then it could be `parataxis:shared`, but not extended to subordinate clauses.

In any case, I've implemented this in 55ee81b.
